### ISSUE TITLE
salon

### ISSUE THREAD

**Issue by MartinDelille** [MartinDelille/libomf#1]
This issue is only made for general discussion.

**Comment by MartinDelille** [comment]
@trebmuh This is a work in progress, but maybe you have knowledge about this format? Its would allow Audacity and maybe Ardour to support a common exchange format with proprietary tools like P** **s.

**Comment by trebmuh** [comment]
Hi @MartinDelille ,
Thanks for pinging me here.
I've got no real knowledge about that format beyond that "it looks to be an open format for music sharing".
So, you're idea is about making a lib of it, then other software can depend on that lib to be able to import/export rather than reinventing the wheel in every software, right?

**Comment by MartinDelille** [comment]
exactly!

**Comment by trebmuh** [comment]
Nice thinking. I'll try to get some knowledge around that.
Will share those here if I've got some.

**Comment by MartinDelille** [comment]
I'm currently working on it from a piece of code I found. I should publish something soon but any input would help!

I'm also studying the [AAF](https://en.wikipedia.org/wiki/Advanced_Authoring_Format).

**Comment by trebmuh** [comment]
[Read that](https://discourse.ardour.org/t/aaf/87442/29) (_where Paul is the man who made Ardour and Jack_).

It looks that Reaper started to work on a something like that (in 2009) under a BSD license, and a first effort started to [get it included in Ardour then](https://github.com/Ardour/ardour/tree/master/tools/omf).

I'd highly suggest you to go on #lad and/or #ardour IRC channels on freenode.net to discuss the issue with knowledgeable people before starting coding things.

Hope that helps.

**Comment by MartinDelille** [comment]
You rock @trebmuh ! I'll have a look at it tomorrow.

**Comment by trebmuh** [comment]
You're welcome. Let me know how that goes.

**Comment by Kirktopode** [comment]
@MartinDelille Is this dead? I just ran into a month or two of free time and I was trying to find a library or some other first steps to convert OMF files. If you know anything about OMF reading/conversion beyond what I'm seeing here, it would be super helpful.

**Comment by MartinDelille** [comment]
Yes it is dead. I finally used this library which work for audio session interchanged using AAF: https://aaf.sourceforge.net/
